src/RuleSet.php

```diff
@@ -322,7 +320,7 @@ final class RuleSet implements RuleSetInterface
         ],
         '@PHPUnit50Migration:risky' => [
             '@PHPUnit48Migration:risky' => true,
-            'php_unit_dedicate_assert' => ['target' => PhpUnitTargetVersion::VERSION_5_0],
```

Comment:
please revert
this is 50migration ruleset, and default target will not be 5_0 on 3.0

Reply:
How to modify test? Exclude all `@PHPUnitXXMigration` sets?

Reply:
not really. need to think about best approach here

Reply:
So let's [ensure](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4433) the rule set are targeting proper PHPUnit's version.